build/legacy-mongo-shell/test.example.js

```diff
@@ -0,0 +1,29 @@
+// Move to test.js and run with `task mongo-test`.
+
+(function() {
+    "use strict";
+
+    const col = db.test;
+
+    col.drop();
+
+    const init = [
```

Comment:
I would rename them `docs`.

Reply:
Well, `expected` and `actual` are docs too